WordPress/Classes/ViewRelated/Jetpack/Branding/Banner/JetpackBannerView.swift

```diff
@@ -29,3 +26,32 @@ class JetpackBannerView: UIView {
     private static let jetpackBannerBackgroundColor = UIColor(light: .muriel(color: .jetpackGreen, .shade0),
                                                               dark: .muriel(color: .jetpackGreen, .shade90))
 }
+
+// MARK: Responding to scroll events
+extension JetpackBannerView: Subscriber {
+
+    typealias Input = CGFloat
+    typealias Failure = Never
+
+    func receive(subscription: Subscription) {
+        subscription.request(.unlimited)
+    }
+
+    func receive(_ input: CGFloat) -> Subscribers.Demand {
+
+        let isHidden: Bool = input < 0
+
+        guard self.isHidden != isHidden else {
+            return .unlimited
+        }
+
+        UIView.animate(withDuration: 0.3, delay: 0, options: [.curveEaseIn], animations: { [weak self] in
+            self?.isHidden = isHidden
+        }, completion: { _ in
+
```

Comment:
Shouldn't we pass nil to completion here?

Reply:
good catch @alpavanoglu ! Indeed we should